src/components/DailyGrowth/DailyGrowth.jsx

```diff
@@ -172,30 +224,35 @@ export default function DailyGrowth() {
                     }}
                     cursor={{ fill: theme.colors.backgroundTertiary }}
                   />
-                  <Bar
-                    name={t('confirmedCasesShort')}
-                    stackId={STACK_PER_DATE_ID}
-                    dataKey={DAILY_CASES_KEY}
-                    fill={theme.colors.negative}
-                  />
-                  <Bar
-                    name={t('deaths')}
-                    stackId={STACK_PER_DATE_ID}
-                    dataKey={DAILY_DEATH_KEY}
-                    fill={theme.colors.primary}
-                  />
-                  <Bar
-                    name={t('cured')}
-                    stackId={STACK_PER_DATE_ID}
-                    dataKey={DAILY_CURES_KEY}
-                    fill={theme.colors.positive}
-                  />
+
+                  {barGraph.filter(filterGroups).map((item) => (
+                    <Bar
+                      name={item.name}
+                      stackId={STACK_PER_DATE_ID}
+                      dataKey={item.key}
+                      fill={item.color}
+                    />
+                  ))}
                 </BarChart>
               </ResponsiveContainer>
             </StyledBody>
           </ResponsiveTab>
         </Tabs>
       )}
+      <div className='daily-growth-filter'>
+        {graphFilter.map((item, index) => (
+          <label>
+            <input
+              name={item.key}
+              type='checkbox'
+              dataKey={item.key}
+              checked={item.selected}
+              onChange={() => handleInputChange(item, index)}
+            />
+            {item.key}
+          </label>
```

Comment:
There is a [Checkbox ](https://baseweb.design/components/checkbox/) component in the UI library we're using. It'll look better imho. :)